src/SprykerEco/Yves/Computop/Plugin/PayuCeeSingleSubFormPlugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Plugin;
```

Comment:
* Change to the `SprykerEco\Yves\Computop\Plugin\CheckoutPage`.
* The folder in which the plugin is created must have the name of the module where it will be installed.

Reply:
Done